README.md

```diff
@@ -1,41 +1,40 @@
 <!--
-SPDX-FileCopyrightText: 2022 German Aerospace Center (DLR)
+SPDX-FileCopyrightText: 2022 German Aerospace Center (DLR), Forschungszentrum Jülich
 
 SPDX-License-Identifier: CC-BY-SA-4.0
 -->
 
 <!--
 SPDX-FileContributor: Stephan Druskat
 SPDX-FileContributor: Michael Meinel
+SPDX-FileContributor: Oliver Bertuch
 -->
 
+![HERMES Key Visual](docs/source/_static/img/opengraph-workflow.png)
+
```

Comment:
I see. Will create a new version  which is more compact and even smaller in size.

Reply:
Done. Now 1200x350px instead of OpenGraph 1200x630px. And now 270KB instead of 720KB in size.

![](https://github.com/hermes-hmc/workflow/raw/feature/35-write-first-tutorials/docs/source/_static/img/header.png)